src/Token/AccessTokenInterface.php

```diff
@@ -69,4 +70,20 @@ public function __toString();
      * @return array
      */
     public function jsonSerialize();
+
+    /**
+     * Sets the clock.
+     *
+     * @param Clock $clock a clock.
+     *
+     * @return void
+     */
+    public function setClock(Clock $clock);
+
+    /**
+     * Get the current time, whether real or simulated.
+     *
+     * @return int
+     */
+    public function getTimeNow();
```

Comment:
Both of these methods added to the interface represent a BC break.

Reply:
right, of course